### PR TITLE
my first station project

### PR DESCRIPTION
**Pull request by Gug888**
Users simply input their text into the provided text area and select the desired analysis option (count letters, count words, or count sentences). The analyzer then processes the text and displays the corresponding count.We have button that change background color.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var rooot = document.getElementById('root');
```
Comment (Arpine-javakhkuhi) [inline script.js:1]: Please, change  'var' into modern js -  'let' or 'const'

File: script.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+var rooot = document.getElementById('root');
+var parentInp = document.createElement('input');
+var firstButton = document.createElement('button');
+var secondButton = document.createElement('button');
+var textMessage = document.createElement('p');
+var randomBgButton = document.createElement('button')
+var lastButton = document.createElement('button')
+parentInp.classList.add('texts');
```
Comment (Arpine-javakhkuhi) [inline script.js:8]: Adding a class will not be needed if you write HTML in a proper file with needed CSS selector.

File: script.js (hunk 1 above)
Comment (Arpine-javakhkuhi) [reply script.js:1]: 'rooot' or 'root'? 
You can add extension if you use Visual Studio for checking words speller - 'Code Spell Checker'.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+lastButton.classList.add('sentences')
+secondButton.innerText = 'Words Count';
+lastButton.innerText = 'Sentences Count'
+randomBgButton.innerText = 'Generate Background';
+randomBgButton.classList.add('bgButton')
+document.body.append(rooot)
+rooot.append(parentInp, firstButton, secondButton, lastButton, randomBgButton, textMessage)
+firstButton.onclick = function () {
+    var val = parentInp.value.replace(/[^a-zA-Z]/g, "");
+    return textMessage.innerText = 'The texts letters count is' + ' ' + val.length
```
Comment (Arpine-javakhkuhi) [inline script.js:21]: There is no need to return in onclick function.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+};
+secondButton.onclick = function () {
+    var inp = parentInp.value.trim()
+    var word = inp.split(/\s+/).filter(el => el !== "");
+
+    return textMessage.innerText = 'In entered text words counts are' + ' ' + word.length;
```
Comment (Arpine-javakhkuhi) [inline script.js:27]: There is no need to return in onclick function.

File: script.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+};
+lastButton.onclick = function () {
+    var sent = parentInp.value.trim()
+    var sentence = sent.split(/[.!?]+/).filter(elem => elem !== "");
+    return textMessage.innerText = 'The sentence count are' + ' ' + sentence.length
```
Comment (Arpine-javakhkuhi) [inline script.js:32]: There is no need to return in onclick function.

File: script.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+}
+randomBgButton.onclick = function () {
+    var r = Math.floor(Math.random() * 255);
+    var g = Math.floor(Math.random() * 255);
+    var b = Math.floor(Math.random() * 255);
+    document.body.style.backgroundColor = `rgb(${r},${g},${b})`
+    console.log(r, g, b);
```
Comment (Arpine-javakhkuhi) [inline script.js:39]: Please, remove console.log

File: script.js (hunk 1 above)
Comment (Gug888) [reply script.js:1]: I made the necessary changes you mentioned, in new branch.
Thank you!

**Comment by gaboyan** [comment]
The website is really cool, but I couldn't write a multi-line text in the textarea to check how it counts sentences in such texts.
It counts the number of sentences incorrectly when the text contains URLs or email addresses.

BTW, repos name is not as it should be :)

**Pull request merged**